src/OrchardCore/OrchardCore.ReCaptcha.Core/TagHelpers/ReCaptchaTagHelper.cs

```diff
@@ -21,22 +23,23 @@ public class ReCaptchaTagHelper : TagHelper
         private readonly IHttpContextAccessor _httpContextAccessor;
         private readonly ReCaptchaSettings _settings;
         private readonly ILogger<ReCaptchaTagHelper> _logger;
+        private readonly ILocalizationService _localizationService;
 
-        public ReCaptchaTagHelper(IOptions<ReCaptchaSettings> optionsAccessor, IResourceManager resourceManager, IHttpContextAccessor httpContextAccessor, ILogger<ReCaptchaTagHelper> logger)
+        public ReCaptchaTagHelper(IOptions<ReCaptchaSettings> optionsAccessor, IResourceManager resourceManager, ILocalizationService localizationService, IHttpContextAccessor httpContextAccessor, ILogger<ReCaptchaTagHelper> logger)
         {
             _resourceManager = resourceManager;
             _httpContextAccessor = httpContextAccessor;
             _settings = optionsAccessor.Value;
             Mode = ReCaptchaMode.PreventRobots;
             _logger = logger;
+            _localizationService = localizationService;
         }
 
         [HtmlAttributeName("mode")]
         public ReCaptchaMode Mode { get; set; }
 
         public override void Process(TagHelperContext context, TagHelperOutput output)
         {
```

Comment:
Here you could use the async version => `ProcessAsync`
So that below you will not have to use a `GetAwaiter()`

Reply:
Done